assignment/scripts/describe.js

```diff
@@ -136,6 +136,9 @@ let mix = true;
 
 if (mix === true) {
   colorOne = 'purple';
+
+// FIX - the script fails to set colorTwo to 'purple' since mix === True will evaluate to true.
```

Comment:
You are right that colorTwo is missing from the if-block, but that is not the only mistake in the code. What else is wrong?